templates/services/merge/distributedValidator/charon.tmpl

```diff
@@ -0,0 +1,24 @@
+{{/* charon.tmpl */}}
+{{ define "distributedValidator" }}
+  charon:
```

Comment:
```suggestion
  dv:
```

Should we consider picking a container name that allows for us to swap in a future nethermind DV client without renaming things away from `charon`? (Not an important suggestion if this doesn't impact how DVs are addressed by downstream validator clients.

Reply:
Fixed!